### ISSUE TITLE
Literal Integer, asList(), returns a NullPointer instead of single item list.

### ISSUE THREAD

**Issue by hudsonmendes** [bxfsh/boxfish-commons-web-model#9]
Assume we have a model like the following:
```java
final RestModel query = newRestModel()
    .permit("labelIds")
    .value("labelIds", 4817);
```
Then we attempt to retrieve the value as a list of Integer:
```java
List<Integer> labelIds = query.get("labelIds").asListOf(Integer.class);
```
This fails, because get("labelIds").asList() fails internally, returning a null pointer.

**Literals should all become a single item list on demand, whenever we request them.
And that should be it for _all_ literal values.**

**Comment by hudsonmendes** [comment]
This issue was closed in the PRs #14  and #15 version 1.4 and is currently working in production.

**Comment by hudsonmendes** [comment]
This issue was closed in the PRs #14  and #15 version 1.4 and is currently working in production.
